src/overlays/actors/ovl_Bg_Goron_Oyu/z_bg_goron_oyu.c

```diff
@@ -2,8 +2,6 @@
 
 #define FLAGS 0x00000030
 
-#define THIS ((BgGoronOyu*)thisx)
-
```

Comment:
From what I understand, the usage of this/thisx may not be permanent and what it will end up being is still open to discussion. But for consistency of what other actors are doing atm and also so it'll be easier to change all the actors simultaneously, my thoughts is that it would be better to leave in `#define THIS ((BgGoronOyu*)thisx)` and use it where applicable